src/Cake.Npm/NpmSettingsExtensions.cs

```diff
@@ -48,5 +48,51 @@ public static NpmSettings FromPath(this NpmSettings settings, DirectoryPath path
 
             return settings;
         }
+
+        /// <summary>
+        /// Sets the StandardError-Action
+        /// </summary>
+        /// <param name="settings">The settings.</param>
+        /// <param name="standardErrorAction">The StandardError-Action.</param>
+        /// <returns>The <paramref name="settings"/> instance with <see cref="Core.Tooling.ToolSettings.WorkingDirectory"/> set to <paramref name="path"/>.</returns>
+        public static NpmSettings WithStandardError(this NpmSettings settings, Action<string> standardErrorAction)
+        {
+            if (settings == null)
+            {
+                throw new ArgumentNullException(nameof(settings));
+            }
+
+            if (standardErrorAction == null)
+            {
+                throw new ArgumentNullException(nameof(standardErrorAction));
+            }
+
+            settings.StandardErrorAction = standardErrorAction;
+
+            return settings;
+        }
+
+        /// <summary>
+        /// Sets the StandardOutput-Action
+        /// </summary>
+        /// <param name="settings">The settings.</param>
+        /// <param name="standardOutputAction">The StandardOutput-Action.</param>
+        /// <returns>The <paramref name="settings"/> instance with <see cref="Core.Tooling.ToolSettings.WorkingDirectory"/> set to <paramref name="path"/>.</returns>
+        public static NpmSettings WithStandardOutput(this NpmSettings settings, Action<string> standardOutputAction)
```

Comment:
I would suggest to follow [naming in Cake.Core](https://github.com/cake-build/cake/blob/develop/src/Cake.Core/Extensions/ProcessSettingsExtensions.cs#L69): `SetStandardOutputHandler`

Reply:
changed.